challenge3/webapp/tasukujp/app/models/energy.rb

```diff
@@ -0,0 +1,57 @@
+class Energy < ApplicationRecord
+
+  belongs_to :house
+
+  validates :label, :house_id, :year, :month, :temperature, :daylight, :energy_production, presence: true
+  validates :house_id, uniqueness: { scope: %i(year month) }
+
+  class << self
+    def csv_header_converters
+      lambda do |name|
+        {
+            ID:               :id,
+            Label:            :label,
+            House:            :house_id,
+            Year:             :year,
+            Month:            :month,
+            Temperature:      :temperature,
+            Daylight:         :daylight,
+            EnergyProduction: :energy_production
+        }[name.intern] || raise
+      end
+    end
+
+    def csv_converters
+    end
+
+    def sum_graph_data(year:, month:)
+      _year = year.present? ? year : 2013 #Time.current.year
+      _month = month.present? ? month : 6 #Time.current.month
+      result = Energy.where(year: _year, month: _month).joins(:house).group('houses.city').sum(:energy_production)
+      {year: _year, month: _month}.merge(report: result)
```

Comment:
単純に
```suggestion
      {year: _year, month: _month, report: result}
```
と書けばよかったのではないか。
何かダメな理由があったのかな。